package.json

```diff
@@ -51,7 +53,7 @@
       "git add"
     ]
   },
-  "main": "lib/index.js",
+  "main": "lib/src/index.js",
```

Comment:
That most definitely doesn't look right.

Reply:
Fixed with https://github.com/ridedott/eslint-plugin/pull/30/commits/0d1b41b2e6fa7c18ddca54619c872d55e5350a02